### DISCUSSION TITLE
git files corrupted

### DISCUSSION THREAD

**Discussion by mtyminsky** [Homebrew/discussions#4065] [Getting started]
Hi!
I wanted to install homebrew, but I cannot download index from a mirror.

Error lines below:
error: Unable to get pack index https://mirrors.sustech.edu.cn/homebrew/homebrew-core.git/objects/pack/pack-80ef12b400c3622c6892186d49c4651e6664d990.idx
error: Unable to get pack index https://mirrors.sustech.edu.cn/homebrew/homebrew-core.git/objects/pack/pack-b8a93aceda0cdd8d638cd9cfc90dac74540dbf89.idx

**Comment by p-linnane** [comment]
You will need to reach out to whoever maintains that mirror. We cannot provide any support here.
